app/main.py

```diff
@@ -1,3 +1,2 @@
-# TODO: add initial code
-def hello_world():
-    return "Hello, world!"
+if __name__ == '__main__':
+    pass
```

Comment:
Here you can make the same, as was in example:
- import everything
- first line: CinemaHall.objects = CinemaHallManager()
- add some examples of how to call methods.
And make this code commented :)

Reply:
Also delete tests folder

Reply:
Done